package.json

```diff
@@ -37,7 +37,7 @@
     "fix:yarn-deduplicate": "yarn install && yarn-deduplicate --strategy=fewer && yarn install",
     "lint": "npm-run-all --continue-on-error \"lint:*\"",
     "@TODO.2": "Upgrade or remove these blocks and remove the --ignore-package options (also @TODO.1)",
-    "lint:dependency-version-consistency": "check-dependency-version-consistency . --ignore-dep=@blockprotocol/graph --ignore-dep=@blockprotocol/type-system --ignore-package=@apps/hashdotdev --ignore-package=@blocks/chart --ignore-package=@blocks/embed --ignore-package=@blocks/person",
+    "lint:dependency-version-consistency": "check-dependency-version-consistency . --ignore-dep=@blockprotocol/graph --ignore-package=@apps/hashdotdev --ignore-package=@blocks/embed --ignore-package=@blocks/person",
```

Comment:
The same change needs to be made to `fix:`

Reply:
Whoops, thanks for flagging! [`22b41a4` (#4646)](https://github.com/hashintel/hash/pull/4646/commits/22b41a4d6992454bfaac53ceeb7017b06fd9cb0f)